src/js/view/controls/components/timeslider.ts

```diff
@@ -183,24 +209,24 @@ class TimeSlider extends Slider {
             this.updateCues(null, cues);
         }
 
-        const tracks = playlistItem.tracks;
-        each(tracks, function (track) {
+        const tracks: TextTrackList = playlistItem.tracks;
+        each(tracks, function (this: TimeSliderWithMixins, track: TextTrackLike): void {
             if (track && track.kind && track.kind.toLowerCase() === 'thumbnails') {
-                this.loadThumbnails(track.file);
+                this.loadThumbnails(track.file as string);
             } else if (track && track.kind && track.kind.toLowerCase() === 'chapters') {
-                this.loadChapters(track.file);
+                this.loadChapters(track.file as string);
             }
         }, this);
     }
 
-    performSeek() {
-        const percent = this.seekTo;
+    performSeek(): void {
+        const percent = this.seekTo as number;
         const duration = this._model.get('duration');
         let position;
         if (duration === 0) {
             this._api.play(reasonInteraction());
         } else if (this.streamType === 'DVR') {
-            const seekRange = this._model.get('seekRange') || { start: 0 };
+            const seekRange = { start: 0 };
```

Comment:
updated here @robwalch given dead code

Reply:
This is actually a bug Joe. `seekRange` should be set. I don't know at what point it was removed. We haven't been testing DVR playback adequately. If you setup the player with a live stream (moving window) and change `minDvrWindow` to be smaller than the live stream's window, it will play in DVR window. The longer the steam plays (and the more the window moved) the more seeking is impacted.

Please put back the `'seekRange'` removals. I can DM you a test case and we can open an issue if necessry.